.tsbuildinfo

```diff
@@ -0,0 +1 @@
:2531},{"file":"./lib/destack/src/lib/css/index.ts","start":747,"length":30,"messageText":"Object is possibly 'null'.","category":1,"code":2531},{"file":"./lib/destack/src/lib/css/index.ts","start":804,"length":33,"messageText":"Object is possibly 'null'.","category":1,"code":2531},{"file":"./lib/destack/src/lib/css/index.ts","start":838,"length":5,"code":2339,"category":1,"messageText":"Property 'style' does not exist on type 'Element'."}]],209,137,[213,[{"file":"./lib/destack/src/lib/traits234/index.ts","start":557,"length":4,"code":2322,"category":1,"messageText":"Type 'string' is not assignable to type 'never'."},{"file":"./lib/destack/src/lib/traits234/index.ts","start":585,"length":10,"code":2322,"category":1,"messageText":"Type 'any' is not assignable to type 'never'."},{"file":"./lib/destack/src/lib/traits234/index.ts","start":645,"length":5,"code":2322,"category":1,"messageText":"Type 'any' is not assignable to type 'never'."},{"file":"./lib/destack/src/lib/traits234/index.ts","start":1247,"length":26,"code":2345,"category":1,"messageText":"Argument of type 'string' is not assignable to parameter of type 'never'."}]],140,118,214,103,107,105,106,108,104,100,48,50,101,51,52,53,54,55,56,57,58,59,60,61,62,63,64,49,97,65,66,67,102,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,99,86,90,91,92,93,94,98,95,96,215,216,122,113,129,115,114,109,112,116,117,127,126,124,123,125,120,121,110,111,130,119,131,128,132,133,87,88,89,10,11,13,12,2,14,15,16,17,18,19,20,21,3,4,25,22,23,24,26,27,28,5,29,30,31,32,6,33,34,35,36,7,41,37,38,39,40,8,42,43,44,1,9,45,46],"affectedFilesPendingEmit":[[134,1],[136,1],[135,1],[47,1],[210,1],[207,1],[211,1],[141,1],[142,1],[143,1],[144,1],[145,1],[146,1],[147,1],[148,1],[149,1],[150,1],[151,1],[152,1],[153,1],[154,1],[155,1],[156,1],[157,1],[158,1],[159,1],[160,1],[161,1],[162,1],[163,1],[164,1],[165,1],[166,1],[167,1],[168,1],[169,1],[170,1],[171,1],[172,1],[173,1],[174,1],[175,1],[176,1],[177,1],[178,1],[179,1],[180,1],[181,1],[182,1],[183,1],[184,1],[185,1],[186,1],[187,1],[188,1],[189,1],[190,1],[191,1],[192,1],[193,1],[194,1],[195,1],[196,1],[197,1],[198,1],[199,1],[200,1],[201,1],[202,1],[203,1],[204,1],[212,1],[206,1],[205,1],[138,1],[139,1],[208,1],[209,1],[137,1],[217,1],[213,1],[140,1],[118,1],[214,1],[103,1],[107,1],[105,1],[106,1],[108,1],[104,1],[100,1],[48,1],[50,1],[101,1],[51,1],[52,1],[53,1],[54,1],[55,1],[56,1],[57,1],[58,1],[59,1],[60,1],[61,1],[62,1],[63,1],[64,1],[49,1],[97,1],[65,1],[66,1],[67,1],[102,1],[68,1],[69,1],[70,1],[71,1],[72,1],[73,1],[74,1],[75,1],[76,1],[77,1],[78,1],[79,1],[80,1],[81,1],[82,1],[83,1],[84,1],[85,1],[99,1],[86,1],[90,1],[91,1],[92,1],[93,1],[94,1],[98,1],[95,1],[96,1],[215,1],[216,1],[122,1],[113,1],[129,1],[115,1],[114,1],[109,1],[112,1],[116,1],[117,1],[127,1],[126,1],[124,1],[123,1],[125,1],[120,1],[121,1],[110,1],[111,1],[130,1],[119,1],[131,1],[128,1],[132,1],[133,1],[87,1],[88,1],[89,1],[2,1],[3,1],[4,1],[5,1],[6,1],[7,1],[8,1],[9,1],[46,1]]},"version":"4.3.2"}
```

Comment:
Not sure why this stayed here since it's added to `.gitignore`.

Maybe this would help: [https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore](https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore)